cumulusci/tasks/release_notes/generator.py

```diff
@@ -12,6 +12,7 @@
 class BaseReleaseNotesGenerator(object):
     def __init__(self):
         self.change_notes = []
+        self.empty_pull_requests = []
```

Comment:
Thinking about the naming here, we should probably call this `empty_change_notes` for symmetry with the existing `change_notes` list. Pull requests are the source of the change notes in the case of the github release notes generator, but this base class doesn't assume that's where they come from.